packages/react-select/src/components/LiveRegion.tsx

```diff
@@ -112,7 +113,7 @@ const LiveRegion = <
     const isSelected = !!(
       focusedOption &&
       selectValue &&
-      selectValue.includes(focusedOption)
+      selectValue.some(option => getOptionValue(option.value) === getOptionValue(focusedOption))
```

Comment:
`... getOptionValue(option) ...`

Reply:
Arg, yes sorry, shouldn't make these quick changes without thinking about it